components/ratgdo/__init__.py

```diff
@@ -84,11 +84,11 @@ async def to_code(config):
 
     cg.add_library(
         name="secplus",
-        repository="https://github.com/esphome-ratgdo/secplus",
-        version="f98c3220356c27717a25102c0b35815ebbd26ccc",
+        repository="https://github.com/esphome-ratgdo/secplus#f98c3220356c27717a25102c0b35815ebbd26ccc",
+        version=None,
```

Comment:
We might as well update this one as well.

Added perms to https://github.com/ratgdo/secplus as well